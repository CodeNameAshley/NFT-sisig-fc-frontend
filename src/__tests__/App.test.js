import { cleanup, render } from "@testing-library/react";
import App from "../components/App";
import { unmountComponentAtNode } from "react-dom";

let container = null


beforeEach(() => {
  // setup a DOM element as a render target
  container = document.createElement("div");
  document.body.appendChild(container);
});

afterEach(() => {
  // cleanup on exiting
  unmountComponentAtNode(container);
  container.remove();
  container = null;
});

it("Should take a snapshot", () => {
  const {asFragment} = render(<App />);
  
  expect(asFragment(<App />)).toMatchSnapshot()
});