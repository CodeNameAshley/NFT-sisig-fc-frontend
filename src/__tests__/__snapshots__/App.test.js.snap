// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <h1>
      SFC
    </h1>
  </div>
</DocumentFragment>
`;
